index.html

```diff
@@ -522,10 +728,17 @@ <h3 class="contact-me-header text-align">CONTACT US</h3>
             <button
               type="submit"
               aria-label="see project"
-              class="see-project-button border-none form-button"
+              class="see-project-button border-none display-none-lg"
             >
               See Projects
             </button>
+            <button
+              type="submit"
+              aria-label="see project"
+              class="see-project-button see-project-button-form border-none display-none-sm display-block-lg"
+            >
+              Send
+            </button>
           </div>
         </form>
```

Comment:
- [ ] Please, consider the design to be closer to Figma one. Here you can see on your version the space between the inputs.
- [ ] Please, consider the spaces between icons in the footer
> You version
<img width="541" alt="Screen Shot 2022-02-17 at 14 15 48" src="https://user-images.githubusercontent.com/49064106/154455791-526d7a16-2718-4d75-bd2c-c329d6f68643.png">

> Figma design
<img width="376" alt="Screen Shot 2022-02-17 at 14 17 01" src="https://user-images.githubusercontent.com/49064106/154456824-5a9e9474-9a69-403e-902d-7573ea72ce6f.png">